packages/webpack/__e2e-fixtures__/typescript/app.ts

```diff
@@ -1,4 +1 @@
-declare var module : any
-declare var process : any
-
 module.exports = process.env.TEST
```

Comment:
I'd like to keep having something TypeScript specific in the `app.ts` - currently we set the webpack config to read from `app.ts` and have the ts block set up, but I think it would also work without, as it currently is valid JS.

Probably a simple 
```ts
module.exports = process.env.TEST as string
```
would be enough

Reply:
@zcei What about this?

```ts
function add(a: number, b: number): number {
  return a + b;
}
```

Reply:
Even better